WordPressVIPMinimum/Sniffs/UserExperience/AdminBarRemovalSniff.php

```diff
@@ -307,14 +303,12 @@ public function process_text_for_style( $stackPtr, $file_name ) {
 				if ( false !== strpos( $content, $property ) ) {
 					$error = true;
 
-					if ( true === $this->remove_only ) {
-						// Check the value of the CSS property.
-						if ( preg_match( '`' . preg_quote( $property, '`' ) . '\s*:\s*(.+?)\s*(?:!important)?;`', $content, $matches ) > 0 ) {
-							$value = trim( $matches[1] );
-							$valid = $this->validate_css_property_value( $value, $requirements['type'], $requirements['value'] );
-							if ( true === $valid ) {
-								$error = false;
-							}
+					// Check the value of the CSS property.
+					if ( ( true === $this->remove_only ) && preg_match( '`' . preg_quote( $property, '`' ) . '\s*:\s*(.+?)\s*(?:!important)?;`', $content, $matches ) > 0 ) {
```

Comment:
Should we remove the extra set of brackets enclosing `true === $this->remove_only`?